commit f97a36967e34b750f33b59802be4dfaf174efaee (refs/changes/61/2761/2)
Author: Trent Mick <trentm@gmail.com>
Date:   2017-10-10T14:18:43-07:00 (2 years ago)
    
    TOOLS-1883 sdcadm shouldn't hardcode path to psql
    Reviewed by: Cody Peter Mello <cody.mello@joyent.com>
    Approved by: Cody Peter Mello <cody.mello@joyent.com>

diff --git a/lib/common.js b/lib/common.js
index 71180e6..cce8d49 100644
--- a/lib/common.js
+++ b/lib/common.js
@@ -5,7 +5,7 @@
  */
 
 /*
- * Copyright 2017 Joyent, Inc.
+ * Copyright (c) 2017, Joyent, Inc.
  */
 
 var assert = require('assert-plus');
@@ -996,7 +996,10 @@ function waitForPostgresUp(opts, cb) {
             '-n',
             opts.server,
             /* JSSTYLED */
-            format('/usr/sbin/zlogin %s "/opt/local/bin/psql -U postgres -t -A -c \'SELECT NOW() AS when;\'"', opts.vm)
+            format('/usr/sbin/zlogin %s '
+                + '\'PATH=/opt/postgresql/current/bin:/opt/local/bin:$PATH '
+                + 'psql -U postgres -t -A -c "SELECT NOW() AS when;"\'',
+                opts.vm)
         ];
 
         execFilePlus({
